src/index.js

```diff
@@ -73,7 +74,16 @@ function setBuildNumber(usePackageJsonVersion, branch, gitHubToken, gitHubRepo,
 			if(!data.mergeable) {
 				console.error(`Pull request #${ pullRequestId } is not mergeable into master.`);
 				process.exit(1);
-			} else {
+			}
+			else if (!nameMatchesConvention(enforceNamingConvention, data.head.ref)){
+				console.error(`Branch name '${ data.head.ref }' does not match naming convention: '${ namingConventionRegex }'`);
```

Comment:
We can use `branch` instead of `data.head.ref` as this has already been sanitised